code/src/components/thoughts.css

```diff
@@ -0,0 +1,39 @@
+
+.thoughts__container {
+    background: #fff;
+    grid-column: span 3;
+    display: grid;
+    grid-template-columns: 1fr;
+}
+
+.thoughts__card {
+    background: #fff;
+    border: 2px solid #969696;
+    box-shadow: 5px 5px;
+    justify-self: center;
+    padding: 10px;
+    margin: 5px;
+    width: 80%;
+}
+
+.thoughts__info-container, .thoughts__message-and-like {
```

Comment:
No need for separate class names if the styling is the same. Also .thoughts__message-and-like is never used